index.html

```diff
@@ -8,22 +8,30 @@
     <link rel="stylesheet" href="css/styles.css" type="text/css" />
   </head>
   <body>
-    <h1>Learn to code by watching others</h1>
 
-    <p>See how experienced developers solve problems in real-time. Watching scripted tutorials is great, 
-    but understanding how developers think is invaluable. </p>
-  
-    <div>Try it free 7 days then $20/mo. thereafter</div>
-    <div>
-      First Name 
-      Last Name 
-      Email Address
-      Password
-  
-      <div>Claim your free trial </div>
+    <div class="text-blurb">
+      <h1>Learn to code by watching others</h1>
+      <p>See how experienced developers solve problems in real-time. Watching scripted tutorials is great, 
+      but understanding how developers think is invaluable. </p>
+    </div>
+
+    <div class="try-it">
+      <p><strong>Try it free 7 days</strong> then $20/mo. thereafter</p>
     </div>
+
+    <div class="sign-up">
+      <div class="sign-box">First Name</div>
+      <div class="sign-box">Last Name</div>
+      <div class="sign-box">Email Address</div>
+      <div class="sign-box">Password</div>
   
-    By clicking the button, you are agreeing to our Terms and Services
+      <div class="claim-trial"> 
+        <p>Claim your free trial</p>
+      </div>
+
+      <h4> By clicking the button, you are agreeing to our Terms and Services </h4>
```

Comment:
You can also use an input here. If you wrap it with a `label` then clicking the label will also trigger the input.

```
<label><input type="checkbox />By clicking the button, you are agreeing to our Terms and Services</label>

Reply:
There isn't a checkbox here in the example given. It does look like the "Terms and Services" text looks like a hyperlink.
I'll check the requirements again.

Reply:
Gotcha